src/alfasim_score/units.py

```diff
@@ -10,5 +10,5 @@
 ROUGHNESS_UNIT = "mm"
 STD_VOLUMETRIC_FLOW_RATE_UNIT = "sm3/d"
 MASS_FLOW_RATE_UNIT = "kg/s"
-PRESSURE = "kgf/cm2"
-TEMPERATURE = "degC"
+PRESSURE_UNIT = "kgf/cm2"
```

Comment:
I just figured it out now, but I believe the score json pressure unit is psi.

Reply:
Thanks! Fixed.